docs/source/quickstart.rst

```diff
@@ -288,46 +288,205 @@ So far we have been talking about generators that generate one column in isolati
+   |                       0.0, 70.8026416053, 106.6802133604, Lithograph                       |                                                                           [None, -18.456152268697725, 2.330663099603008, None]                                                                          |
+   |           81.2802, 543.5611, 424.1808, Painted cast iron, glazed lava, and glass           |                                                                                  [None, None, None, 'Pencil on vellum']                                                                                 |
+   |                          12.3825, 41.2751, 41.2751, Crumpled map                           |                                                                   [None, 24.543223703363182, 20.0212206104858, 'Gelatin silver print']                                                                  |
+   |                 1.1, 14.0, 18.0, Plastic box containing nine offset cards                  |                                                                      [None, None, None, 'Gelatin silver printing-out-paper print']                                                                      |
+   |                  0.0, 38.1000762002, 45.7200914402, Etching with aquatint                  |                                                                                [None, None, None, 'Gelatin silver print']                                                                               |
+   |          0.0, 25.6, 25.9, One from a set of six records with lithographic sleeves          |                                                                         [None, 95.70986740244891, 72.83297376657666, 'Etching']                                                                         |
+   +--------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   (artwork.depth_cm,width_cm,height_cm,medium)
+
+In the listing above I have had to cut the amount of source statistics reported from the database because datafaker actually produces far more than this!
+As we have five columns, this implies up to 32 missingness patterns.
+Currently, none of our generators suppress the generation of data from patterns that are not represented in the source database at time of configuration;
+therefore data for all 32 possible are pulled into the source stats file.
+This works well if we want future missingness patters to be accounted for,
+but it does mean that our Information Governance people have to look at a huge amount of summary statistics
```

Comment:
IG themselves won't be looking at the summary stats as part of the routine workflow, they approve the process